setup.py

```diff
@@ -15,7 +15,7 @@ def read(filename):
     python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
     install_requires=['Django>=1.11'],
     extras_require={
-        'azure': ['azure>=3.0.0', 'azure-storage-blob>=1.3.1'],
+        'azure': ['azure-common', 'azure-storage-blob>=1.3.1'],
```

Comment:
`azure-common` should be removed. `azure-storage-blob` already depends on it (so it will install it)

Reply:
True

Reply:
I took this into consideration in #684